test/bench/ok/palindrome.drun-run.ok

```diff
@@ -1,9 +1,9 @@
 ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
 ingress Completed: Reply: 0x4449444c0000
-debug.print: (true, +1_188, 12_869)
-debug.print: (false, +1_188, 12_199)
-debug.print: (false, +1_188, 12_859)
-debug.print: (true, +868, 12_460)
-debug.print: (false, +868, 11_404)
-debug.print: (false, +868, 12_424)
+debug.print: (true, +1_188, 13_066)
```

Comment:
this is a slight increase, no idea why it happens (considering the seen size decrease of the Wasm).